src/app/screens/LNURLWithdraw/index.tsx

```diff
@@ -47,19 +47,24 @@ function LNURLWithdraw() {
     try {
       setErrorMessage("");
       setLoadingConfirm(true);
+
       const invoice = await api.makeInvoice({
         amount: parseInt(valueSat),
         memo: details.defaultDescription,
       });
 
-      await axios.get(details.callback, {
+      const response = await axios.get(details.callback, {
         params: {
           k1: details.k1,
           pr: invoice.paymentRequest,
         },
       });
 
-      setSuccessMessage("Withdraw request sent successfully.");
+      if (response.data.status === "OK") {
```

Comment:
I'd do a .toUpperCase() here. 
what if a service returns `status: "ok"`

```suggestion
      if (response.data.status && response.data.status.toUpperCase() === "OK") {
```

Reply:
Hm, according to the docs of `axios` the `status` is always a number.

Either check for `statusText` (not recommended, see docs)
or check for `status > 200` for GET requests

https://axios-http.com/docs/res_schema

Reply:
Thanks, updated. Guess I'm more used to consistent repsonses :P